transcom/mymove
# INT B-20452 - Review and submit button bug fix

## [Agility ticket](tbd)

## Summary

This is to fix a bug I've found for my code change in this PR: https://github.com/transcom/mymove/pull/13666

The issue is that the 'Review and submit' button is disabled when only HHG, NTS, or NTS-release is created and going back to the move page before signing to complete(see screenshot).

This issue is due to the function I've wrote to check if shipments all shipments are completed - but this has flaws where it did not account for HHG, NTS, and NTS-release and giving false value.

### How to test

1. Create a move as a customer and create a HHG shipment, but do not sign to complete the move.
2. Go back to the main page by clicking 'Finish later'.
3. Verify 'Review and submit' button is not disabled.
4. Do the same for NTS and NTS-release and verify the button is not disabled.
5. Add a boat shipment and only fill out 1st page of the form and go back to the main page so that the shipment is set to incomplete.
6. Verify that 'Review and submit' button is disabled.
7. Edit the boat shipment to complete it and verify that you are able to continue to the page to sign and complete the move.

## Screenshots
### ISSUE:
![image](https://github.com/user-attachments/assets/8fa56e9f-dd40-4173-8ddd-eb65f196943f)

### FIXED:
![image](https://github.com/user-attachments/assets/5a2c3ea6-496c-4d24-80db-9106ad4ffce0)
![image](https://github.com/user-attachments/assets/9239f2c1-f9d9-4812-86f7-b3a3eb410cc4)
